plugins/package-managers/cocoapods/src/funTest/kotlin/CocoaPodsFunTest.kt

```diff
@@ -57,10 +55,7 @@ class CocoaPodsFunTest : WordSpec({
 
             val result = create("CocoaPods").resolveSingleProject(definitionFile)
 
-            result.replaceIssueTimestamps().toYaml() shouldBe patchExpectedResult(expectedResultFile, definitionFile)
+            result.withInvariantIssues().toYaml() shouldBe patchExpectedResult(expectedResultFile, definitionFile)
```

Comment:
This does more than necessary, because in contrast to NPM, CocoaPods does provide a deterministic ordering of issues. Would it be worth providing a rationale in the commit message?